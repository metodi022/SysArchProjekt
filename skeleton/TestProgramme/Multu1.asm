lui $1,0x0000
lui $2,0x0000
ori $1,0x000a
addiu $2, $2, 0x0003
multu $1, $2
mflo $31
mfhi $30