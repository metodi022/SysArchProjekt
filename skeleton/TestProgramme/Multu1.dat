0x3c010000
0x3c020000
0x3421000a
0x24420003
0x00220019
0x0000f812
0x0000f010
