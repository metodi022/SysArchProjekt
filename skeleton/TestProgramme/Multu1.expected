@1 a
@2 3
@1e 0
@1f 1e